gui.py

```diff
@@ -81,18 +82,14 @@ def start_simulation(self):
         """
         Function to start the simulation by running the UVSim.
         """
-        self.uvsim.cpu.run(self.uvsim.mem, self.uvsim.io_device)
-        self.update_main_control_frame()
 
-        # self.simulation_running = True
-        # self.run_simulation_step()
+        self.simulation_running = True
+        self.run_simulation_step()
         
     def run_simulation_step(self):
         if self.simulation_running and not self.uvsim.cpu.waiting_for_input:
```

Comment:
There are quite a few conflicts with `main` currently. If you can resolve those and get this working that would be great! =)

Steps to get to the resolution:
- `git checkout main`
- `git pull`
- `git checkout feature/InputFix`
- `git merge main`

This will produce an error saying there are conflicts. You can view what files conflict with:

- `git status`

You'll need to edit the files to resolve the conflict. Make sure that you can still run the application normally when you're done. Let me know if you would like a second pair of eyes on the conflicts.